lnrpc/routerrpc/router_backend.go

```diff
@@ -1,18 +1,27 @@
 package routerrpc
 
 import (
+	"bytes"
 	"encoding/hex"
 	"errors"
 	"fmt"
+	"time"
 
+	"github.com/btcsuite/btcd/chaincfg"
 	"github.com/btcsuite/btcutil"
+	"github.com/lightningnetwork/lnd/invoices"
 	"github.com/lightningnetwork/lnd/lnrpc"
 	"github.com/lightningnetwork/lnd/lnwire"
 	"github.com/lightningnetwork/lnd/routing"
 	"github.com/lightningnetwork/lnd/routing/route"
+	"github.com/lightningnetwork/lnd/zpay32"
 	context "golang.org/x/net/context"
 )
 
+var (
+	zeroHash [32]byte
```

Comment:
can use `lntypes.Hash{}`

Reply:
Maybe we should even define a constant `lntypes.ZeroHash`? :)

Reply:
good idea, created